### PR TITLE
delapp: Add version 1.0.2

### PR DESCRIPTION
**Pull request by entr0pia**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/delapp.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "version":  "1.0.1.2",
```
Comment (StarsbySea) [inline bucket/delapp.json:2]:
```suggestion
    "version": "1.0.2",
```

File: bucket/delapp.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "description": "A simple tool to delete files or folders in Windows",
+    "homepage":  "https://github.com/differentrain/Delapp",
+    "license":  "MIT",
+    "url":  "https://github.com/differentrain/Delapp/releases/download/1.0.1/Delapp.zip",
```
Comment (StarsbySea) [inline bucket/delapp.json:6]:
```suggestion
    "url": "https://github.com/differentrain/Delapp/releases/download/1.0.2/Delapp.zip",
```

**Review by StarsbySea — CHANGES_REQUESTED** [review]

File: bucket/delapp.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "1.0.2",
+    "description": "A simple tool to delete files or folders in Windows",
+    "homepage":  "https://github.com/differentrain/Delapp",
+    "license":  "MIT",
+    "url": "https://github.com/differentrain/Delapp/releases/download/1.0.2/Delapp.zip",
+    "hash":  "7bc05877fa4b2dbe8fe0c5104b5e641fcfae9aefad74ef62d3aa61afb10f9d79",
```
Comment (StarsbySea) [inline bucket/delapp.json:7]:
```suggestion
    "hash": "e1613df86a3a4928b36100de1412af694cf84476e655b12c8f50ef9560c17227",
```

**Review by StarsbySea — CHANGES_REQUESTED** [review]

**Review by StarsbySea — APPROVED** [review]

**Comment by HUMORCE** [comment]
https://github.com/differentrain/Delapp#requirement

suggests `extras/windowsdesktop-runtime` or `extras/windowsdesktop-runtime-lts`

```
    "suggest": {
        ".NET Desktop Runtime": "extras/windowsdesktop-runtime",
    },
```

File: bucket/delapp.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "1.0.2",
+    "description": "A simple tool to delete files or folders in Windows",
+    "homepage":  "https://github.com/differentrain/Delapp",
+    "license":  "MIT",
```
Comment (aliesbelik) [inline bucket/delapp.json:5]:
```suggestion
    "license":  "MIT",
    "suggest": {
        ".NET Desktop Runtime": "extras/windowsdesktop-runtime",
    },
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]

**Review by aliesbelik — APPROVED** [review]

**Pull request merged**